src/lib.rs

```diff
@@ -17,19 +17,20 @@
 //! devices and send events to the virtual devices.
 //! Virtual devices are created in `/sys/devices/virtual/input`.
 //!
-//! Devices emit events, represented by the [`InputEvent`] type. Each device supports a few different
+//! Devices emit events, represented by the [`EvdevEvent`] trait. Each device supports a few different
```

Comment:
there's a principle of naming convention being violated here imo... more to say below...